### ISSUE TITLE
Rats! WebGL hit a snag.

### ISSUE THREAD

**Issue by azurewraith** [stockto2/desktophero_django#49]
Hi,

Some time after I started logging in I noticed DTH would stop loading in Chrome...

![image](https://user-images.githubusercontent.com/287646/38932547-8c33e3c8-42e4-11e8-9775-9dfc8716415b.png)

Here's what my chrome console reported:

![image](https://user-images.githubusercontent.com/287646/38932601-b32a8856-42e4-11e8-87b3-0d41f1517c99.png)

It works in a different browser, but after I tried to login and submit an asset it started to go awry until I logged out and deleted all cookies/cache globally.  Doing the same in Chrome didn't help but I only could delete any storage/cache related to desktophero3d.com.

**Comment by azurewraith** [comment]
@stockto2 - any ideas?

**Comment by stockto2** [comment]
Hey, sorry I missed this. This is a problem with Chrome or possibly with your computer not being able to handle the graphics. Is your Chrome updated to the latest version?

**Comment by azurewraith** [comment]
Yeah, my Chrome is (and has been) updated properly.  It is currently at Version 67.0.3396.87.  I just checked and it still wasn't working (console error was the same) although no "Rats! WebGL hit a snag".  I had it actually work once today when I fired up the tab w/ the inspector up, but a subsequent refresh it went back to broken.

My computer is an i7 MacBook Pro w/ 16 GB of RAM.  DTH used to work fine in Chrome and at one point didn't.  Right now it's working in Safari for me, so that's what I've been using.
